pkg/models/devops/devops.go

```diff
@@ -36,7 +36,7 @@ func init() {
 	flag.StringVar(&JenkinsUrl, "jenkins-url", "http://ks-jenkins.kubesphere-devops-system.svc.cluster.local:80", "jenkins server host")
 }
 
-func GetPipeline(projectName, pipelineName string, req *http.Request) (*Pipeline, error) {
+func GetPipeline(projectName, pipelineName string, req *http.Request) (*TypePipeline, error) {
```

Comment:
what does `TypePipeline` and `RunPipeline` mean? confusing here

Reply:
`Typepipeline` is Pipeline struct,and `RunPipeline` mean runs in a pipeline, such as https://github.com/jenkinsci/blueocean-plugin/tree/master/blueocean-rest#get-all-runs-in-a-pipeline.

Reply:
I will update this struct name.